src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -279,6 +280,11 @@ public function setRow($column, array $row)
      */
     public function render()
     {
+        if ($this->output instanceof ConsoleSectionOutput) {
+            // TODO: Calculate the number of lines to remove for the table
```

Comment:
outdated todo?

Reply:
No this still needs to be done, currently it's only calculating the number of rows and headers, but not taking the row separator into account